amundsen_application/static/js/components/TableDetail/OwnerEditor/index.tsx

```diff
@@ -240,7 +267,19 @@ export class OwnerEditor extends React.Component<OwnerEditorProps, OwnerEditorSt
 export const mapStateToProps = (state: GlobalState) => {
   const ownerObj = state.tableMetadata.tableOwners.owners;
   const items = Object.keys(ownerObj).reduce((obj, ownerId) => {
-    obj[ownerId] = { label: ownerObj[ownerId].display_name }
+    const { profile_url, user_id, display_name } = ownerObj[ownerId];
+    let profileLink;
+    let isExternalLink;
+    if (AppConfig.indexUsers.enabled) {
```

Comment:
I believe this misses the case where `indexUser.enabled == false` and `profileLink` should still link to an external profile_url URL. 

```
let profileLink = profile_url;
let isExternalLink = true;
if (AppConfig.indexUsers.enabled) {
  isExternalLink = false;
  profileLink = `/user/...`;
}

```

Reply:
Okay, I thought we shouldn't display profile links if user indexing disabled. Fixed